packages/json-viewer/src/JsonViewer.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import classnames from 'classnames'
+
+export interface JsonViewerProps {
+  data: Record<string, unknown>;
+  expandAll?: boolean;
+  listClassNames?: string | string[];
+  keyClassNames?: string | string[];
+  summaryClassNames?: string | string[];
+}
+
+type Detailable = Omit<JsonViewerProps, 'listClassNames'>;
+
+function isPrimitive(value: unknown): value is string | number | boolean {
+  return ['string', 'number', 'boolean'].includes(typeof value);
+}
+function isObject(value: unknown): value is Record<string, unknown> {
+  return !!value && typeof value === 'object';
+}
+
+function getDetails({ data, expandAll, keyClassNames, summaryClassNames }: Detailable): (JSX.Element | null)[] {
+  return Object.entries(data).map(entry => {
+    const [key, value] = entry;
+    if (isPrimitive(value)) {
+      return (
+        <li key={key} className="pt-1">
+          <span className={classnames('text-bold', keyClassNames)}>
+            {`${key}:`}
+          </span>
+          <span> {(value as string | number | boolean).toString()}</span>
```

Comment:
I don't think you need the `as string...` part here since you have the `isPrimitive` check